diff --git a/node_modules/react-native-css-interop/.cache/android.js b/node_modules/react-native-css-interop/.cache/android.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/ios.js b/node_modules/react-native-css-interop/.cache/ios.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/macos.js b/node_modules/react-native-css-interop/.cache/macos.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/native.js b/node_modules/react-native-css-interop/.cache/native.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/windows.js b/node_modules/react-native-css-interop/.cache/windows.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js b/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js
index 59ea66e..da3525a 100644
--- a/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js
+++ b/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js
@@ -1757,10 +1757,14 @@ function parseDisplay(display, options) {
     }
 }
 function parseAspectRatio(aspectRatio) {
+    if (!aspectRatio) return "auto";
     if (aspectRatio.auto) {
         return "auto";
     }
     else {
+        if (!aspectRatio.ratio || !Array.isArray(aspectRatio.ratio) || aspectRatio.ratio.length < 2) {
+            return "auto";
+        }
         if (aspectRatio.ratio[0] === aspectRatio.ratio[1]) {
             return 1;
         }
diff --git a/node_modules/react-native-css-interop/src/css-to-rn/parseDeclaration.ts b/node_modules/react-native-css-interop/src/css-to-rn/parseDeclaration.ts
index bf095a5..45aadba 100644
--- a/node_modules/react-native-css-interop/src/css-to-rn/parseDeclaration.ts
+++ b/node_modules/react-native-css-interop/src/css-to-rn/parseDeclaration.ts
@@ -2639,9 +2639,13 @@ function parseAspectRatio(
   // This is missing types
   aspectRatio: any,
 ): RuntimeValueDescriptor {
+  if (!aspectRatio) return "auto";
   if (aspectRatio.auto) {
     return "auto";
   } else {
+    if (!aspectRatio.ratio || !Array.isArray(aspectRatio.ratio) || aspectRatio.ratio.length < 2) {
+      return "auto";
+    }
     if (aspectRatio.ratio[0] === aspectRatio.ratio[1]) {
       return 1;
     } else {
